src/request.rs

```diff
@@ -42,6 +44,9 @@ pub struct Request {
     /// Disable introspection queries for this request.
     #[serde(skip)]
     pub disable_introspection: bool,
+
+    #[serde(skip)]
+    pub(crate) parsed_query: Option<ExecutableDocument>,
```

Comment:
This change is actually semver breaking. Since code that would've directly contructed this public struct would not compile anymore.

@sunli829 thoughts?

Reply:
Thanks a lot, I just realized this, so I created the `async-graphql-v4` branch. 😁

Reply:
oh... that's really unfortunate
I don't think we should allow `Request` to be constructed from fields in future.
It is just pain in ass to worry about compatibility